examples/connor-python-example/TransactionsBetweenUsers.py

```diff
@@ -0,0 +1,438 @@
+import random
+import datetime
+import traceback
+import asyncio
+from gremlin_python.process.graph_traversal import __
+from gremlin_python.process.traversal import T, Direction
+from gremlin_python.process.anonymous_traversal import traversal
+from gremlin_python.process.traversal import P
+from gremlin_python.driver.driver_remote_connection import DriverRemoteConnection
+from dash import Dash, html
+import dash_cytoscape as cyto
+from dash import dcc
+from dash.dependencies import Input, Output, State
+import dash_bootstrap_components as dbc
+
+
+
+HOST = "localhost"
+PORT = 8182
+
+def create_cluster():
+    return DriverRemoteConnection("ws://localhost:8182/gremlin", "g")
+
+def print_all_elements(g):
```

Comment:
Nit mostly for python formatting suggests 2 lines between functions
```suggestion
PORT = 8182


def create_cluster():
    return DriverRemoteConnection("ws://localhost:8182/gremlin", "g")


def print_all_elements(g):
```